### DISCUSSION TITLE
Camera Orientation

### DISCUSSION THREAD

**Discussion by sg000** [cameroncros/OctoPrint-DiscordRemote#255] [Q&A] [answered]
With the new Octoprint/Octopi version that moved camera settings into a plugin, now my discordremote snapshots are not orientated properly. Haven't figured out how to resolve it. Octolapse I just modified the cam settings, but we don't have that in discordremote. Suggestions?

**Comment by cameroncros** [comment]
I have seen some discussion of this on the octoprint discord. Its probably that I am not getting the camera settings properly. I am probably affected as well. If I get time I will look into it, but I am also happy if anyone else wants to submit a PR to fix it.

**Comment by cameroncros** [comment]
Turns out it is even easier to get the snapshot with the new api. The only complication is how to pick which camera is used for the snapshot, I have some code to iterate over all the available plugins until I get a valid image, and then stop. Will probably create a release with that for now, and picking the correct camera can happen later.

**Comment by cameroncros** [comment, marked as answer]
And it turns out it isn't that easy, but I think I have gotten the camera api working now.

See release v5.9

**Reply by sg000** [reply]
Thanks. downloading update now.
